agent/internal/testutil/api.go

```diff
@@ -0,0 +1,26 @@
+package testutil
```

Comment:
I realize the original file this was taken from is called api, but could you name this such that it's a general-purpose place to put utility functions for agent tests (assuming that's what you intended this package for)?

Reply:
What do you mean by this? I feel like we usually call our test helper function packages "testutil".

Reply:
What do you mean by this? I thought we usually call our test helper function packages "testutil".

Reply:
Sorry I meant the file name. It's named api but doesn't have anything api-related really

Reply:
Ah, gotcha. I updated the file name.